plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLUtils.java

```diff
@@ -221,6 +221,21 @@ public static String makeSQLLike(String like)
         return like.replace("*", "%").replace("?", "_");
     }
 
+    public static String makeGlobFromSqlLikePattern(String sqlLikePattern) {
```

Comment:
Maybe `makeGlobFromLike`? Also, please specify nullability annotations. They are essential in our codebase.

Reply:
I guess to have a more specific function name is more readable for such big namespace of the class. And second one, why do we need it if IDEA provide default annotation from the box 
![image](https://github.com/dbeaver/dbeaver/assets/43766501/1c422b0c-7346-41c9-8b7c-4698b7a5e739)

Maybe we don't need it anymore?

Reply:
Not everyone uses IntelliJ as their preferred IDE. Moreover, it's possible to generate compile-time checks for these which is impossible without explicit annotations.

Reply:
The class itself is called `SQLUtils,` suggesting that its methods are somehow related to SQL. If you look at other methods here, you will notice that they don't contain the term "SQL" in their names.

Therefore, it should be called `makeGlobFromLike` to match other methods.

Reply:
Actually, I see a number of methods with SQL in their name in the class. Also such util method is avaliable in almost entire codebase of our, so it's better to have a name exhaustive name for such method. Moreover, you are always able to use static import, what brings confusing to the name without SQLUtils.

Reply:
Static imports are forbidden in our code base.